client/dom/types/colorScale.ts

```diff
@@ -29,6 +24,7 @@ export type ColorScaleOpts = {
 	/** Optional. Shows a value in the color bar for the default, bottom axis
 	 * This value cannot be zero at initialization.*/
 	markedValue?: number
+	numericInputs?: NumericInputs
```

Comment:
please document purpose

Reply:
A comment is added in https://github.com/stjude/proteinpaint/pull/2517/commits/6b87a51393dc5b1214b811b71ba6e7e6a22c886f.